packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -336,17 +341,30 @@ export class Datepicker implements ComponentInterface {
   });
 
   private setValidState(value: string): void {
-    let formattedDate: string;
-    let parsedDate: Date;
+    if(this.inoRange) {
+      let arr = value.match(/(\d[^a-z]+)/g).map(match => this.validateInput(match.trim()));
```

Comment:
Could you extract that regex as a static member and give it a name? This way, it is more clear what the regex is supposed to test.